frontend/src/components/AvailabilitySelector/index.js

```diff
@@ -44,8 +44,8 @@ export default function AvailabilitySelector({ timeAvailability, onChange }) {
 
   // if null or invalid, treat as 9-5 local time
   if (timeAvailability === null || timeAvailability.length !== 24) {
-    timeAvailability = localToUtc('NNNNNNNNNYYYYYYYYNNNNNNN');
-    onChange(timeAvailability);
+    console.error(`Invalid timeAvailability value ${timeAvailability}; substituting default`);
+    timeAvailability = AvailabilitySelector.default();
```

Comment:
It turns out that calling `onChange` during a render is a React no-no.  So I added the `default` method and handled this invalid case by reporting the error and "pretending" that the supplied property was the default value.

Reply:
I'm a little confused as to why you'd want to do that. Does `onChange` do something special here? And in `onRangeChange` and in `onCustomChange` and `onCustomChange`, couldn't you just return the `localToUTC` value instead of calling `onChange`?  To me it looks like onChange -> onChange is happening.

Reply:
I realized you're passing a different function to `onChange`, but it threw me because I thought this was some new way of using reacts build-in inline `onChange` event handler. :)